docs/portal/image-registry/managing-repositories-under-a-project.md

```diff
@@ -0,0 +1,12 @@
+# Managing Repositories Under a Project
+
+A project can have one or more repositories and a repository can have one or more tags. 
+
+The following procedure describes how to view repositories under a project, how to change a repository's description, and how to delete a repository.
+
+1. In the left pane, click **Image Registry > Registry Projects**.
+2. On the Registry Projects page, click the name of a project. The Edit page appears. This page has two sections:<ul><li>**Basic information**  shows read-only information about the selected project.</ul></li><ul><li>**Repositories** shows the repositories associated with the selected project. A field at the top left allows you to filter the repositories shown.</ul></li>
```

Comment:
Please change "Edit page" to "detail page"

Reply:
Fixed